src/css/theme/_search-box.scss

```diff
@@ -0,0 +1,30 @@
+.ais-search-box {
+  display: inline-block;
```

Comment:
I'll try to hack something with :after and :before but these wont be button for submit or/and reset like ris since it would involve new options to the widget.